gradle/libs.versions.toml

```
@@ -9,7 +9,7 @@ lifecycleRuntimeKtx = "2.9.3"
 activityCompose = "1.10.1"
 composeBom = "2024.09.00"
 
-blueBillywig = "8.32.0"
+blueBillywig = "8.33.0"
```

```suggestion
blueBillywig = "8.33-2-SNAPSHOT"
```

this together with the revert from the comment above should be sufficient, so we keep the external libraries declaration (url + version) in libs.versions.toml